js/render-random-offers.js

```diff
@@ -0,0 +1,52 @@
+import {getRandomOffers} from './data.js';
+
+const offerCardTemplate = document.querySelector('#card')
+  .content
+  .querySelector('.popup');
+
+
+const mapCanvas = document.querySelector('.map__canvas');
+const randomOffers = getRandomOffers();
+const randomOffersFragment = document.createDocumentFragment();
+
+randomOffers.forEach(({author, offer}) => {
+  const offerCard = offerCardTemplate.cloneNode(true);
+  const popupPhoto = offerCard.querySelector('.popup__photo');
+  const popupPhotos = offerCard.querySelector('.popup__photos');
+  let offerType;
+
+  offerCard.querySelector('.popup__title').textContent = offer.title;
+  offerCard.querySelector('.popup__text--address').textContent = offer.address;
+  offerCard.querySelector('.popup__text--price').textContent = `${offer.price} ₽/ночь`;
+
+  if (offer.type == 'palace') {
+    offerType = 'Дворец';
+  }
+  if (offer.type == 'flat') {
+    offerType = 'Квартира';
+  }
+  if (offer.type == 'house') {
+    offerType = 'Дом';
+  }
+  if (offer.type == 'bungalow') {
+    offerType = 'Бунгало';
+  }
+
+  offerCard.querySelector('.popup__type').textContent = offerType;
```

Comment:
Для таких кейсов удобнее использовать референтный объект типа:
```js
const ROOM_TYPES_MAP = {
  palace: 'Дворец',
  flat: 'Квартира',
  ...
}
```
```suggestion
  offerCard.querySelector('.popup__type').textContent = ROOM_TYPES_MAP[offer.type];
```